src/wallet/rpcwallet.cpp

```diff
@@ -467,7 +467,7 @@ Value listaddressgroupings(const Array& params, bool fHelp)
             "[\n"
             "  [\n"
             "    [\n"
-            "      \"bitcoinaddress\",     (string) The bitcoin address\n"
+            "      \"zcashaddress\",     (string) The zcash address\n"
             "      amount,                 (numeric) The amount in btc\n"
             "      \"account\"             (string, optional) The account (DEPRECATED)\n"
```

Comment:
This is the result, so it will always be "", i.e.
```
"      \"zcashaddress\",         (string) The Zcash address\n"
"      amount,                 (numeric) The amount in ZEC\n"
"      \"\",                     (string) The account (always \"\")\n"
```